dnsimple/domains.go

```diff
@@ -28,6 +28,18 @@ type Domain struct {
 	UpdatedAt    string `json:"updated_at,omitempty"`
 }
 
+// DomainListOptions specifies the optional parameters you can provide
+// to customize the DomainsService.ListDomains method.
+type DomainListOptions struct {
```

Comment:
@weppos this is related to `DomainsService.ListDomains` and the API method is `listDomains`, shall we name this `DomainsListOptions`?